cpp/oneapi/dal/table/backend/register_serializable.cpp

```diff
@@ -16,13 +16,16 @@
 
 #include "oneapi/dal/table/backend/empty_table_impl.hpp"
 #include "oneapi/dal/table/backend/homogen_table_impl.hpp"
+#include "oneapi/dal/table/backend/heterogen_table_impl.hpp"
 #include "oneapi/dal/table/backend/csr_table_impl.hpp"
 
 using oneapi::dal::backend::empty_table_impl;
 using oneapi::dal::backend::homogen_table_impl;
+//using oneapi::dal::backend::heterogen_table_impl;
```

Comment:
can remove line completely if not using, same with line 29

Reply:
I'll add FIXME comment instead

Reply:
fixed